docs/under-the-hood/storage.md

```diff
@@ -1,138 +1,222 @@
+
+- `SkipListNode<LabelIndex::Entry>` object has **24B**.
+- `SkipListNode<LabelPropertyIndex::Entry>` has at least **80B**.
+- Each `SkipListNode` has an additional **16B** because of the **next_pointers**.
 
 ### Properties
 
-Nodes and relationships can store various properties. Properties are similar to
-mappings or tables containing property names and their accompanying values.
-Property names are represented as text, while values can be of different types.
-Each property can store a single value, and it is not possible to have multiple
-properties with the same name on a single graph element. But, the same property
-names can be found across multiple graph elements. Also, there are no
-restrictions on the number of properties that can be stored in a single graph
-element. The only restriction is that the values must be of the supported types.
-Below is a table of supported data types.
-
- Type      | Description
------------|------------
- `Null`    | Property has no value, which is the same as if the property does not exist.
- `String`  | A character string (text).
- `Boolean` | A boolean value, either `true` or `false`.
- `Integer` | An integer number.
- `Float`   | A floating-point number (real number).
- `List`    | A list containing any number of property values of any supported type under a single property name.
- `Map`     | A mapping of string keys to values of any supported type.
- `Duration`| A period of time.
- `Date`    | A date with year, month, and day.
- `LocalTime` | Time within a day without timezone.
- `LocalDateTime` | Date and local time.
-
-Check the reference guide to [read more about temporal
-types](/reference-guide/data-types.md) `Duration`, `Date`, `LocalTime` and
-`LocalDateTime`. 
-
-:::note
-
-Even though it's possible to store `List` and `Map` property values, it is
-impossible to modify them. It is, however, possible to replace them entirely.
-So, the following queries are valid:
-
-```cypher
-CREATE (:Node {property: [1, 2, 3]});
-CREATE (:Node {property: {key: "value"}});
-```
-
-But these are not:
-
-```cypher
-MATCH (n:Node) SET n.property[0] = 0;
-MATCH (n:Node) SET n.property.key = "other value";
-```
+All properties use at least **1B** for metadata - type, size of property ID and
+the size of payload in the case of `NULL` and `BOOLEAN` values, or size of
+payload size indicator (how big is the stored value, for example, integers can
+be 1B, 2B 4B or 8b depending on their value). After the initial byte, some
+properties (for example, those of `TEMPORAL_DATA` or `STRING` values) store
+addition metadata. After metadata, properties store the value. 
 
-:::
+#### Size of the additional metadata and value
+
+|Value            |Size                |Note
+|-----------------|--------------------|
+|`NULL`           |0B                  | The value is written in the first byte of the metadata.
+|`BOOL`           |0B                  | The value is written in the first byte of the metadata.
```

Comment:
```suggestion
|`BOOL`           |1B                  | The value is written in the first byte of the metadata.
```

Reply:
Should `NULL` and `BOOL` be  at least 1B? 1B for metadata? The rest is much more clear now, only Map is missing.